template.env

```diff
@@ -3,5 +3,9 @@ PASSWORD=
 HOST=
 PORT=
 SERVICE=
-PGREST_ENDPOINT=https://atd-postgrest.austinmobility.io/legacy-scripts/traffic_reports
```

Comment:
i had to remove the traffic_reports part of the endpoint, so it would match what the Postgrest class expects. Which means I updated main.py, since it doesn't use the Postgrest class.